pyomo/pysp/scenariotree/server_pyro.py

```diff
@@ -357,7 +356,7 @@ def exec_scenariotreeserver(options):
         #NOTE: this should perhaps be command-line driven, so it can
         #      be disabled if desired.
         print("ScenarioTreeServerPyro aborted. Sending shutdown request.")
-        shutdown_pyro_components(host=options.pyro_host,
+        pyu_pyro.shutdown_pyro_components(host=options.pyro_host,
```

Comment:
Is this a bugfix from another PR that you are sneaking in here?

Reply:
Yup. I noticed that it was simply missing the `pyu_pyro` start to `shutdown_pyro_components`.